terraform/modules/emrfs-lambda/policy-munge-lambda-files/aws_caller.py

```diff
@@ -2,13 +2,36 @@
 
 iam_client = boto3.client('iam')
 rds_data_client = boto3.client('rds-data')
+sts_connection = boto3.client('sts')
 
 """
 ============================================================================================================
 =================================== Helper methods that interact with AWS ==================================
 ============================================================================================================
 """
 
+# assumes mgmt role to set up cognito client associated with mgmt account
+def create_cognito_client(mgmt_account_role_arn):
+    mgmt_account = sts_connection.assume_role(
+        RoleArn=mgmt_account_role_arn,
+        RoleSessionName="mgmt_cognito_rds_sync_lambda"
+    )
+    # create cognito client using the assumed role credentials in mgmt acc
+    global cognito_client
```

Comment:
Not a big fan of global variable assignment inside a function, why not just return the client and call the function from the global scope?